test/decl/subscript/addressors.swift

```diff
@@ -227,3 +227,11 @@ struct RedundantMutableAddressors3 {
     mutableAddressWithNativeOwner { return (someValidAddress(), owner)  } // expected-error {{subscript already has a mutable addressor}}
   }
 }
+
+struct MutableSubscriptInDefaultGetter {
```

Comment:
This file tests addressors specifically -- I think this test case belongs elsewhere.